packages/ui-extensions/docs/surfaces/checkout/reference/targets/purchase.thank-you.cart-line-item.render-after.doc.ts

```diff
@@ -10,9 +10,9 @@ const data: ReferenceEntityTemplateSchema = {
   name: 'purchase.thank-you.cart-line-item.render-after',
   description: `A static extension target that renders on every line item, inside the details under the line item properties element.
 
-    > Caution:
-    > This extension target will not be rendered if the line item is a custom line item belonging to a draft order invoice.
-    `,
+  > Caution:
+  > This extension target will not be rendered if the line item is a custom line item belonging to a draft order invoice.
+  `,
```

Comment:
- [ ] @hheyhhay  made these changes 
- [ ] @hheyhhay aproves these changes to be added to `2024-10` stable release